### ISSUE TITLE
Add checks for invalid binding positions

### ISSUE THREAD

**Issue by justinfagnani** [43081j/eslint-plugin-lit#2]
You can't put a binding in a tag name or attribute name position:

Bad:

```js
html`<${name}></${name}>
```

```js
html`<div ${name}=${value}></div>
```
